src/algebra/module/projective.lean

```diff
@@ -127,11 +128,40 @@ begin
     simp },
 end
 
+/-- A variant of `of_lifting_property'` when we're working over a `[ring R]`,
+which only requires quantifying over modules with an `add_comm_group` instance. -/
+theorem of_lifting_property {R : Type u} [ring R]
```

Comment:
Shouldn't this go in the `ring` section below?

Reply:
Thanks.